src/main/kotlin/ai/digital/integration/server/common/tasks/database/DatabaseStartTask.kt

```diff
@@ -68,13 +71,16 @@ abstract class DatabaseStartTask : DockerComposeUp() {
     @TaskAction
     override fun run() {
         project.logger.lifecycle("Cleaning up previous database containers and networks.")
-        project.exec {
+        execOperations.exec {
             executable = "docker-compose"
-            args = arrayListOf("-f", getDockerComposeFile().path, "down", "--remove-orphans")
+            args = listOf("-f", getDockerComposeFile().path, "down", "--remove-orphans")
+        }
+        execOperations.exec {
+            executable = "docker-compose"
+            args = listOf("-f", getDockerComposeFile().path, "up", "-d")
```

Comment:
why we have docker-compose up command added?

Reply:
This is removed - Because the Docker plugin super.run() , doest have a newer version available, so instead using this up, Note: internally the super.run() has this only 

Because the internal GradleExecUtils still uses project.exec, which we dont accept anymore

@TaskAction
    void run() {
        GradleExecUtils.execWithErrorMessage(project) {
            it.executable "docker-compose"
            it.args "-f", getDockerComposeFile(), "up", "-d"
        }
    }


<img width="1372" height="480" alt="Image" src="https://github.com/user-attachments/assets/e75328bd-5572-4728-8947-d2fbc77181f2" />